### PR TITLE
Add sfsql flag

### PR DESCRIPTION
**Pull request by smtakeda**

### CODE REVIEW

**Comment by smtakeda** [comment]
sorry I did this again.

File: src/henplus/PrintStreamOutputDevice.java [hunk 1]
```diff
@@ -10,57 +12,26 @@
  * The OutputDevice to write to.
  */
 public class PrintStreamOutputDevice implements OutputDevice {
-
     private final PrintStream _outStream;
 
-    public PrintStreamOutputDevice(final PrintStream out) {
```
Comment (bjrke) [inline src/henplus/PrintStreamOutputDevice.java]: why removing final?

File: src/henplus/PasswordEraserThread.java [hunk 2]
```diff
@@ -1,78 +1,85 @@
...
+ * better than nothing, right ?
  */
 class PasswordEraserThread extends Thread {
-
-    private final String _eraser;
-    private boolean _running;
-    private boolean _onHold;
+    private final String eraser;
+    private boolean running;
+    private boolean onHold;
```
Comment (bjrke) [inline src/henplus/PasswordEraserThread.java]: there is no need to change the naming conventions of members

File: src/henplus/EchoCommandProperty.java [hunk 3]
```diff
@@ -1,52 +1,53 @@
...
     }
 
-    @Override
-    public void booleanPropertyChanged(final boolean echoCommands) {
+    public void booleanPropertyChanged(boolean echoCommands) {
         if (echoCommands) {
             _dispatcher.addExecutionListener(this);
-        } else {
+        }
+        else {
```
Comment (bjrke) [inline src/henplus/EchoCommandProperty.java]: I see no benefit of such formating rules

File: src/henplus/AbstractCommand.java [hunk 4]
```diff
@@ -1,101 +1,85 @@
...
-    }
-
-    @Override
-    public String getSynopsis(final String cmd) {
-        return null;
-    }
-
+    public String getShortDescription() { return null; }
+    public String getLongDescription(String cmd) { return null; }
+    public String getSynopsis(String cmd) { return null; }
```
Comment (bjrke) [inline src/henplus/AbstractCommand.java]: i see no benefit of such a reformating

File: src/henplus/AbstractCommand.java [hunk 5]
```diff
@@ -1,101 +1,85 @@
...
-        return null;
-    }
-
-    @Override
-    public String getSynopsis(final String cmd) {
-        return null;
-    }
-
+    public String getShortDescription() { return null; }
+    public String getLongDescription(String cmd) { return null; }
```
Comment (bjrke) [inline src/henplus/AbstractCommand.java]: you changed the behavior of the method

File: src/henplus/SQLMetaDataBuilder.java [hunk 6]
```diff
@@ -56,124 +55,134 @@
...
+            if (_interrupted)
                 return null;
-            }
-
-            final DatabaseMetaData meta = session.getConnection().getMetaData();
 
+            DatabaseMetaData meta = session.getConnection().getMetaData();
+            
             while (tableNamesIter.hasNext() && !_interrupted) {
-                final String tableName = tableNamesIter.next();
```
Comment (bjrke) [inline src/henplus/SQLMetaDataBuilder.java]: there is no benefit in removing final

**Comment by smtakeda** [comment]
Again I'm sorry to bother you. This PR was mistakenly sent. We maintain the changes in our repo and the component is currently internal use only, so we don't need to merge the changes.
Thank you very for reviews and feedbacks anyway!

**Comment by bjrke** [comment]
Its also my fault, I didn't see that it was closed.

**Pull request closed without merging**